backend/api/migrations/0001_initial.py

```diff
@@ -0,0 +1,118 @@
+                ('is_staff', models.BooleanField(default=False, help_text='Designates whether the user can log into this admin site.', verbose_name='staff status')),
+                ('is_active', models.BooleanField(default=True, help_text='Designates whether this user should be treated as active. Unselect this instead of deleting accounts.', verbose_name='active')),
+                ('date_joined', models.DateTimeField(default=django.utils.timezone.now, verbose_name='date joined')),
+                ('user_id', models.AutoField(primary_key=True, serialize=False)),
+                ('age', models.IntegerField(blank=True, null=True)),
+                ('phone_number', models.TextField(blank=True, null=True)),
+                ('birthday', models.DateField()),
+                ('blood_type', models.CharField(blank=True, max_length=2, null=True)),
+                ('height', models.IntegerField(blank=True, null=True)),
+                ('weight', models.IntegerField(blank=True, null=True)),
+                ('groups', models.ManyToManyField(blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of their groups.', related_name='user_set', related_query_name='user', to='auth.group', verbose_name='groups')),
+                ('user_permissions', models.ManyToManyField(blank=True, help_text='Specific permissions for this user.', related_name='user_set', related_query_name='user', to='auth.permission', verbose_name='user permissions')),
+            ],
+            options={
+                'db_table': 'users',
+                'managed': True,
+            },
+            managers=[
+                ('objects', django.contrib.auth.models.UserManager()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='MedicationInfo',
+            fields=[
+                ('medication_id', models.AutoField(primary_key=True, serialize=False)),
+                ('dosage', models.CharField(max_length=8)),
+                ('medication_time', models.JSONField()),
+                ('misc_information', models.CharField(blank=True, max_length=200, null=True)),
+                ('frequency', models.CharField(blank=True, max_length=20, null=True)),
+                ('medication_name', models.CharField(blank=True, max_length=30, null=True)),
+                ('active_medication', models.BooleanField(blank=True, null=True)),
+                ('start_date', models.DateField(blank=True, null=True)),
+                ('end_date', models.DateField(blank=True, null=True)),
+                ('user', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'db_table': 'medication_info',
+                'managed': True,
+            },
+        ),
+        migrations.CreateModel(
+            name='Refills',
```

Comment:
Does this require a new table, or could you instead add the refill information to each medication. That way the UI can make one request to the backend to get the information about the medication and respond back with all the required information.

However, it may be good to have this separate to deal with notifications. If so, disregard the above.